src/EnergyPlus/WindowAC.cc

```diff
@@ -1058,6 +1058,9 @@ namespace WindowAC {
 
             zoneEqSizing.OAVolFlow = windowAC.OutAirVolFlow;
             zoneEqSizing.AirVolFlow = windowAC.MaxAirVolFlow;
+            // Make the Fan be sized by this
+            zoneEqSizing.CoolingAirFlow = true;
+            zoneEqSizing.CoolingAirVolFlow = windowAC.MaxAirVolFlow;
```

Comment:
This will size the Fan to match.

Reply:
https://github.com/NREL/EnergyPlus/blob/f274ccbcef211face33328309188b29b36fd0668/src/EnergyPlus/Autosizing/SystemAirFlowSizing.cc#L237-L238

Reply:
In SystemAirFlowSizing, if CoolingAirVolFlow = windowAC.MaxAirVolFlow (which should be already adjusted?) and dataFracOfAutosizedCoolingAirflow = the target fraction, then how does this get the right answer? I would think this would either be `1 * adjusted air flow`  or  `fraction * design air flow`. This looks like it's `fraction * adjusted air flow`. Maybe windowAC.MaxAirVolFlow is the design air flow which gets multiplied by the fraction. In any event you are getting the right answer now.

Reply:
Apologies, I linked to the wrong bit in SystemAirFlowSizing. The correct bit is:

https://github.com/NREL/EnergyPlus/blob/54f875d7369e64666ce968e42067979efc85f382/src/EnergyPlus/Autosizing/SystemAirFlowSizing.cc#L619-L620

```
Process 2757796 stopped
* thread #1, name = 'energyplus', stop reason = step over
    frame #0: 0x00007ffff50d4370 libenergyplusapi.so.24.2.0`EnergyPlus::SystemAirFlowSizer::size(this=0x00007fffffff4d88, state=0x00007fffffffa980, _originalValue=-99999, errorsFound=0x00007fffffff529f) at SystemAirFlowSizing.cc:620:54
   617 	                    }
   618 	                } else {
   619 	                    if (this->zoneEqSizing(this->curZoneEqNum).CoolingAirFlow && !this->zoneEqSizing(this->curZoneEqNum).HeatingAirFlow) {
-> 620 	                        this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).CoolingAirVolFlow;
   621 	                        if (this->finalZoneSizing(this->curZoneEqNum).CoolDDNum > 0 &&
   622 	                            this->finalZoneSizing(this->curZoneEqNum).CoolDDNum <= state.dataEnvrn->TotDesDays) {
   623 	                            DDNameFanPeak = state.dataWeather->DesDayInput(this->finalZoneSizing(this->curZoneEqNum).CoolDDNum).Title;
(lldb) p this->zoneEqSizing(this->curZoneEqNum).CoolingAirVolFlow
(Real64) 0.28246116346780509
```